app/views/recipes/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h3>Create a new recipe</h3>
+<%= form_with model: @recipe, url: recipes_path, method: :post, local: true do |f| %>
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, :required => "required", autofocus: true, autocomplete: "name" %>
+  </div>
+  <div class="field">
+    <%= f.label :preparation_time %><br />
+    <%= f.text_field :preparation_time, autofocus: true %>
+  </div>
+  <div class="field">
+    <%= f.label :cooking_time %><br />
+    <%= f.text_field :cooking_time, autofocus: true %>
+  </div>
```

Comment:
Consider using `number_field` for these two fields to have one extra layer of validation.